database/benchmarking/benchmarking_suite/src/main/scala/uk/ac/ebi/eva/BenchmarkingSuite/CassandraConnectionParams.scala

```diff
@@ -0,0 +1,18 @@
+package uk.ac.ebi.eva.BenchmarkingSuite
+
+import com.datastax.driver.core.{Cluster, PreparedStatement, Session}
+
+object CassandraConnectionParams {
+    def apply(cluster: Cluster, session: Session, insertStmt: PreparedStatement,
+              blockReadStmt: PreparedStatement) = new CassandraConnectionParams(cluster, session, insertStmt, blockReadStmt)
+}
```

Comment:
Given how many times this is called, it takes more code than the `new` calls. Or is it used implicitly somewhere else?

Reply:
No explicit getters and setters with Scala so this is a necessary boilerplate. I can replace the passed arguments with underscore but that is a bit more cryptic.